src/test/resources/features/resolve/section_args_resolve.feature

```diff
@@ -321,4 +321,31 @@ Scenario Outline: Unresolved index if out of bounds
       | rule       | .get('arg1')       | arg1')    | arg1   |
       | rule       | .get('arg1', None) | arg1',    | arg1   |
       | checkpoint | ['arg2']           | rg2']     | arg2   |
-      | checkpoint | .get('arg1')       | arg1')    | arg1   |
\ No newline at end of file
+      | checkpoint | .get('arg1')       | arg1')    | arg1   |
+
+    Scenario Outline: Resolve index in sections with 'multiext' function
+      Given a snakemake project
+      And I open a file "foo.smk" with text
+      """
+        def additional_func(a, b, c, d):
+         return a + b + c + d
+
+        rule NAME:
+         <data_section>:
+            <line1>,
+            "file",
+            <line3>
+         <exec_section>:
+            <key>
+        """
+      When I put the caret at <signature>
+      Then reference in injection should resolve to "<result>" in "foo.smk"
+      Examples:
+        | data_section | line1                    | line3                    | exec_section | key           | signature | result                   |
+        | input        | multiext("f.", "1", "2") | multiext("f.", "2", "1") | shell        | "{input[1]}"  | 1]}       | multiext("f.", "1", "2") |
```

Comment:
Here is better to resolve not just into `multiext(..)` call, but into it's corresponding agument

Reply:
Done